bergson/collection.py

```diff
@@ -90,6 +94,12 @@ def callback(name: str, g: torch.Tensor):
         dtype=dtype,
         fill_value=0.0,
     )
+    per_doc_scores = torch.full(
```

Comment:
Only support one score per doc, i.e. don't support computing module scores separately for now